src/modules/cmdpal/ext/Microsoft.CmdPal.Ext.TimeDate/Helpers/AvailableResultsList.cs

```diff
@@ -59,9 +59,16 @@ internal static List<AvailableResult> GetList(bool isKeywordSearch, SettingsMana
                 AlternativeSearchTag = ResultHelper.SelectStringFromResources(isSystemDateTime, "Microsoft_plugin_timedate_SearchTagFormat"),
                 IconType = ResultIconType.DateTime,
             },
+            new AvailableResult()
+                {
+                    Value = TimeAndDateHelper.GetNumberOfDayInWeek(dateTimeNow, firstDayOfTheWeek).ToString(CultureInfo.CurrentCulture),
```

Comment:
number of day in week? thought you mife the "calendar week" (week of year) result which makes imo more sense.